zhaquirks/inovelli/VZM31SN.py

```diff
@@ -1,578 +1,12 @@
-                    Ota.cluster_id,
-                    InovelliVZM31SNCluster,
-                ],
-            },
-            2: {
-                PROFILE_ID: zha.PROFILE_ID,
-                DEVICE_TYPE: DeviceType.DIMMER_SWITCH,
-                INPUT_CLUSTERS: [Basic.cluster_id, Identify.cluster_id],
-                OUTPUT_CLUSTERS: [
-                    Identify.cluster_id,
-                    OnOff.cluster_id,
-                    LevelControl.cluster_id,
-                    InovelliVZM31SNCluster,
-                ],
-            },
-        },
-    }
-
-    device_automation_triggers = INOVELLI_AUTOMATION_TRIGGERS
-
-
-class InovelliVZM31SNv9(CustomDevice):
-    """VZM31-SN 2 in 1 Switch/Dimmer Module."""
-
-    signature = {
-        MODELS_INFO: [("Inovelli", "VZM31-SN")],
-        ENDPOINTS: {
-            1: {
-                PROFILE_ID: zha.PROFILE_ID,
-                DEVICE_TYPE: DeviceType.DIMMER_SWITCH,
-                INPUT_CLUSTERS: [
-                    Basic.cluster_id,
-                    Identify.cluster_id,
-                    Groups.cluster_id,
-                    Scenes.cluster_id,
-                    OnOff.cluster_id,
-                    LevelControl.cluster_id,
-                    Metering.cluster_id,
-                    ElectricalMeasurement.cluster_id,
-                    Diagnostic.cluster_id,
-                    INOVELLI_VZM31SN_CLUSTER_ID,
-                    WWAH_CLUSTER_ID,
-                ],
-                OUTPUT_CLUSTERS: [Identify.cluster_id, Ota.cluster_id],
-            },
-            2: {
-                PROFILE_ID: zha.PROFILE_ID,
-                DEVICE_TYPE: DeviceType.DIMMER_SWITCH,
-                INPUT_CLUSTERS: [Identify.cluster_id],
-                OUTPUT_CLUSTERS: [
-                    Identify.cluster_id,
-                    OnOff.cluster_id,
-                    LevelControl.cluster_id,
-                    INOVELLI_VZM31SN_CLUSTER_ID,
-                ],
-            },
-        },
-    }
-
-    replacement = {
-        ENDPOINTS: {
-            1: {
-                PROFILE_ID: zha.PROFILE_ID,
-                DEVICE_TYPE: DeviceType.DIMMABLE_LIGHT,
-                INPUT_CLUSTERS: [
-                    Basic.cluster_id,
-                    Identify.cluster_id,
-                    Groups.cluster_id,
-                    Scenes.cluster_id,
-                    OnOff.cluster_id,
-                    LevelControl.cluster_id,
-                    Metering.cluster_id,
-                    ElectricalMeasurement.cluster_id,
-                    Diagnostic.cluster_id,
-                    InovelliVZM31SNCluster,
-                    WWAH_CLUSTER_ID,
-                ],
-                OUTPUT_CLUSTERS: [
-                    Identify.cluster_id,
-                    OnOff.cluster_id,
-                    LevelControl.cluster_id,
-                    Ota.cluster_id,
-                    InovelliVZM31SNCluster,
-                ],
```

Comment:
`OnOff`, `LevelControl`, and `InovelliVZM31SNCluster` were added for the VZM31-SN (in `InovelliVZM31SNv9`) on ep 1.
Should we also do that in the v2 quirk? Maybe not..?